app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+
+  def get_recent_posts(limit_count = 5)
+    posts.order(created_at: :desc).limit(limit_count)
+  end
```

Comment:
- [ ] Good job working on this but the method should return the 3 most recent posts, so the default value for limit_count should be 3